Assets/MixedRealityToolkit/Definitions/SpatialAwareness/SpatialAwarenessPlanarObject.cs

```diff
@@ -7,16 +7,15 @@ namespace Microsoft.MixedReality.Toolkit.SpatialAwareness
 {
     public partial class SpatialAwarenessPlanarObject : BaseSpatialAwarenessObject
     {
-        // todo
-        public BoxCollider Collider { get; set;  }
+        /// <summary>
+        /// The BoxCollider associated with this plane's GameObject.
+        /// </summary>
+        public BoxCollider Collider { get; set; }
```

Comment:
serialized field private blah blah?

Reply:
Not sure I understand. This is a public property and doesn't need a serialized backing field.

Reply:
This one follows the [Use [...] auto properties where possible](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#access-modifiers) guideline